src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -1039,25 +1039,6 @@ pub const SHM_STAT: ::c_int = 13;
 pub const SHM_INFO: ::c_int = 14;
 pub const SHM_ANON: *mut ::c_char = 1 as *mut ::c_char;
 
-// The *_MAXID constants never should've been used outside of the
-// FreeBSD base system.  And with the exception of CTL_P1003_1B_MAXID,
-// they were all removed in svn r262489.  They remain here for backwards
-// compatibility only, and are scheduled to be removed in libc 1.0.0.
-#[doc(hidden)]
-#[deprecated(since = "0.2.54", note = "Removed in FreeBSD 11")]
-pub const CTL_MAXID: ::c_int = 10;
-#[doc(hidden)]
-#[deprecated(since = "0.2.54", note = "Removed in FreeBSD 11")]
-pub const KERN_MAXID: ::c_int = 38;
-#[doc(hidden)]
-#[deprecated(since = "0.2.54", note = "Removed in FreeBSD 11")]
-pub const HW_MAXID: ::c_int = 13;
-#[doc(hidden)]
-#[deprecated(since = "0.2.54", note = "Removed in FreeBSD 11")]
-pub const USER_MAXID: ::c_int = 21;
-#[doc(hidden)]
-pub const CTL_P1003_1B_MAXID: ::c_int = 26;
```

Comment:
This constant isn't deprecated.

Reply:
But it should be.  And in fact, it has been removed in FreeBSD 13.  How about we remove it in this PR, and I submit another that deprecates it immediately (a non breaking change)?

Reply:
Generally I'd like to avoid a breaking change unless it breaks the build. So, the problem here is that we don't have any policies for supported FreeBSD versions like #1412 IIRC. We have hidden the items and added deprecation notes, I feel it's enough here, isn't it? One possible way is to tweak the deprecation notes like "We plan to remove this in 0.2.7x" so that users notice the removal.
Deprecating `CTL_P1003_1B_MAXID` is welcome, of course.

Reply:
In this case, relying on those constants could cause misbehavior at runtime, which IMHO is worse than a broken build, because it's harder to detect.  There is precedent for removing similar constants which vary from OS release to OS release: 215f095601637ee7b5c73f2d9d5fc67d9a3f843e

Reply:
In 215f095601637ee7b5c73f2d9d5fc67d9a3f843e, we told the user to open an issue in the deprecation notes as I suggested above. But we don't do it yet for `CTL_MAXID` and others.